### DISCUSSION TITLE
track controller

### DISCUSSION THREAD

**Discussion by yanmingrui96** [CoderLine/alphaTab#792] [Q&A] [answered]
Hi author, I'm particularly interested in how to make the kind of track controls that you display on the homepage of your website, such as solo, mute and volume control? Are there any similar tutorials or demos? thanks

**Comment by Danielku15** [comment, marked as answer]
I recommend our main tutorial how to build the foundation of the control on our website: https://alphatab.net/docs/tutorials after that, there should be enough understanding to find the remaining parts on the reference documentation. https://alphatab.net/docs/reference/api
